### ISSUE TITLE
[BUG] haveUserCapabilitiesInDatabase does not correctly set role

### ISSUE THREAD

**Issue by TimothyBJacobs** [lucatume/wp-browser#336]
**Environment**
OS: Mac
PHP version: 7.3
Installed Codeception version: 3.1.1
Installed wp-browser version: 2.2.31
WordPress version: 5.3
Local development environment: Docker
WordPress structure and management: Default

**Codeception configuration file**  

```yaml
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
    commands:
        - Codeception\Command\GenerateWPUnit
        - Codeception\Command\GenerateWPRestApi
        - Codeception\Command\GenerateWPRestController
        - Codeception\Command\GenerateWPRestPostTypeController
        - Codeception\Command\GenerateWPAjax
        - Codeception\Command\GenerateWPCanonical
        - Codeception\Command\GenerateWPXMLRPC
params:
    - tests/codeception.env
```

**Suite configuration file**
```yaml
actor: UpgradeTester
modules:
    enabled:
        - \Helper\Upgrade
        - WPFilesystem
        - WPDb
        - WPCLI
        - Asserts
    config:
        WPDb:
            dsn: 'mysql:host=%TEST_SITE_DB_HOST%;port=%TEST_SITE_DB_PORT%;dbname=%TEST_SITE_DB_NAME%'
            user: '%TEST_SITE_DB_USER%'
            password: '%TEST_SITE_DB_PASSWORD%'
            dump: 'tests/_data/clean.sql'
            #import the dump before the tests; this means the test site database will be repopulated before the tests.
            populate: true
            # re-import the dump between tests; this means the test site database will be repopulated between the tests.
            cleanup: true
            waitlock: 10
            url: '%TEST_SITE_WP_URL%'
            urlReplacement: true #replace the hardcoded dump URL with the one above
            tablePrefix: '%TEST_SITE_TABLE_PREFIX%'
        WPCLI:
            path: '%WP_ROOT_FOLDER%'
            allow-root: true
            user: 1
            env:
                strict-args: true
        WPFilesystem:
            wpRootFolder: '%WP_ROOT_FOLDER%'
        \Helper\Upgrade:
            projectPressAuth: '%PROJECT_PRESS_AUTH%'
            wpRootFolder: '%WP_ROOT_FOLDER%'
```

**Describe the bug**
Subsequent calls to `haveUserCapabilitiesInDatabase` do not overwrite the existing capabilities.

**Output**
```
Failed asserting that two strings are equal.
Expected :'editor'
Actual   :'administrator'
```

**To Reproduce**
```php
public function tryToTest( UpgradeTester $I ) {
	$uid = $I->haveUserInDatabase( 'my-user', 'administrator' );
	$I->haveUserCapabilitiesInDatabase( $uid, 'editor' );

	$I->assertEquals( 'editor', trim( $I->cliToString( [ 'user', 'get', $uid, '--field=roles' ] ) ) );
}
```

**Expected behavior**
The existing capabilities to be overwritten.

**Additional context**
I think this is because it is inserting a _new_ meta row for `wp_capabilities` instead of replacing the existing row. This means when `WP_User` initializing the capabilities with `$single` set to `true` in `get_user_meta` the subsequent rows are not picked up. Inserting a drop call fixes the issue.

```
$I->dontHaveUserMetaInDatabase( [ 'user_id' => $uid1, 'meta_key' => $capsKey ] );
```

**Comment by lucatume** [comment]
@TimothyBJacobs thanks for reporting the issue, this is next on my todo list.

**Comment by lucatume** [comment]
I've fixed this in `2.2.33`; let me know if the issue persists.
